### ISSUE TITLE
RPC authentication failure from 127.0.0.1:62519

### ISSUE THREAD

**Issue by zychappy** [btcsuite/btcd#1404]
here is my conf,I am trying to run a regtest
```
regtest=1
rpcuser=xxx
rpcpass=xxx
txindex=1
notls=1
rpclisten=127.0.0.1:8334
```
when I use curl 
```
curl --user xxx:xxx --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "getinfo", "params": [] }' -H 'content-type: text/plain;' http://127.0.0.1:8334
401 Unauthorized.
```
could anyone tell me how to fix it?

**Comment by Roasbeef** [comment]
You need to use JSON RPC to authenticate, not basic auth.

**Comment by c78867886** [comment]
> You need to use JSON RPC to authenticate, not basic auth.

how to do this? I check the doc and it is really hard to follow since it basically says nothing.
